docs/cel2/decision-tree.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Cel2 Decision Tree
+description: Not sure what you need to do to migrate from Celo L1 to Celo L2?
+---
+
+<details>
+  <summary>I run a node</summary>
+  
+  Follow the [node upgrade instructions](docs/cel2/l2-operator-guide.md).
+
+</details>
+<details>
+  <summary>I use or need a node provider</summary>
+
+  The following node providers will be available shortly after the migration to Cel2:
+
+  - [Quicknode](https://www.quicknode.com/chains/celo) (not yet verified)
+  - [Infura](https://www.infura.io/networks/celo) (not yet verified)
+  - [DSRV](https://drpc.org/chainlist/celo) (not yet verified)
+
+
+  If none of the above work for you, you can use cLabs best effort hosted node: [Forno](https://docs.celo.org/network/node/forno).
+</details>
+<details>
+  <summary>I use ContractKit or Fee Abstraction (ERC-20 gas tokens)</summary>
+
+  Recommended versions for SDK and libraries:
+  - [@celo/connect >=6.1.0](https://www.npmjs.com/package/@celo/connect)
+  - [@celo/contractkit >=8.1.1](https://www.npmjs.com/package/@celo/contractkit)
+  - [@celo/celocli >=5.1.1](https://www.npmjs.com/package/@celo/celocli)
+  - [@viem >=2](https://www.npmjs.com/package/viem)
+  
+  <details>
+  <summary>I need more information on ContractKit...</summary>
+    
+  Have a look at the [ContractKit Cel2 Guide](https://docs.google.com/document/d/1F-9OtZeFOhB7SbgWyUHjOgxBwF-749URZDK0OpgtAqE/edit)
+  
+  </details>
+  
+</details>
+
+<details>
+  <summary>I need help!</summary>
+
+  Reach out to us!
+
+  - Technical question? [celo-org discussions on GitHub](https://github.com/orgs/celo-org/discussions/categories/cel2)
+  - Any type of question? TODO Discord channel
```

Comment:
please add. Please let me know if I should communicate that

Reply:
also add link to Discord

Reply:
The channel didn't exist at the time of writing 🙃 can add as soon as I can link to it